src/main/views/jurisdiction-list.html

```diff
@@ -1,9 +1,8 @@
-
 <div class="form-group">
-       <select id="jurisdictionName" name="jurisdictionName" class="form-control">
-                <option value="" selected>--Select a value--</option>
-                {% for jursidiction in jurisdictions %}
-                    <option value="{{ jursidiction.id }}">{{jursidiction.name}}</option>
-                {%endfor%}
-        </select> 
-</div>     
\ No newline at end of file
+  <select id="jurisdictionName" name="jurisdictionName" class="form-control">
+    <option value="" selected>--Select a value--</option>
+    {% for jurisdiction in jurisdictions %}
+      <option value="{{ jurisdiction.id }}">{{jurisdiction.name}}</option>
+    {% endfor %}
```

Comment:
Should we have space after `{{`

We have {{ jurisdiction.id }} and then {{jurisdiction.name}} on the same line

Reply:
Ideally, there should be :)